erdos/src/lib.rs

```diff
@@ -209,13 +210,12 @@ impl fmt::Display for Uuid {
     }
 }
 
-/// Resets seed and creates a new dataflow graph.
+/// Resets seed.
 pub fn reset() {
     // All global variables should be reset here.
     RNG.with(|rng| {
         *rng.borrow_mut() = StdRng::from_seed(&[1913, 3, 26]);
     });
-    dataflow::graph::default_graph::set(dataflow::graph::AbstractGraph::new());
 }
```

Comment:
We don't need this function anymore, this was being used to reset the default graph.